.gitignore

```diff
@@ -5,5 +5,5 @@ bin
 nri-kubernetes.yaml
 nri-kubernetes.yml
 
-e2e/charts/e2e-resources/Chart.lock
-e2e/charts/e2e-resources/charts/
+charts/e2e-resources/Chart.lock
+charts/e2e-resources/charts/
```

Comment:
It would be awesome that this change had its own commit. Reviewing this would be easier ;)

Reply:
![image](https://user-images.githubusercontent.com/43335750/159679585-2aeaaae9-67fa-4c2e-baa3-0c851288b477.png)